apps/dashboard/src/components/primitives/sonner-helpers.tsx

```diff
@@ -17,3 +17,27 @@ export const showToast = ({
     ...options,
   });
 };
+
+export const showSuccessToast = (message: string, position: 'bottom-center' | 'top-center' = 'bottom-center') => {
```

Comment:
Added helper toast functions to have a consistent success and error toasts.

Reply:
Check if toast.success('Event has been created') will work and be mapped

Reply:
@LetItRock I've checked and the default toast.success is not styled per our design system specs. Wonder if should go in to the rabbit hole, or address globally sonner at a later stage.